packages/back-end/src/models/FeatureModel.ts

```diff
@@ -59,7 +70,10 @@ featureSchema.index({ id: 1, organization: 1 }, { unique: true });
 
 type FeatureDocument = mongoose.Document & LegacyFeatureInterface;
 
-const FeatureModel = mongoose.model<FeatureDocument>("Feature", featureSchema);
+export const FeatureModel = mongoose.model<FeatureDocument>(
```

Comment:
same here. Probably best to keep the mongoose model private and rely on public methods to access it.